packages/create-pantheon-decoupled-kit/src/templates/next-drupal-search-api-addon/components/searchInput.module.css

```diff
@@ -24,23 +24,27 @@
     outline: 0; 
 }
 
+.searchInput:focus { 
+    box-shadow: inset 0 0 0 1px rgb(59,113,202)
+}
+
 .searchBtn {
     display: flex; 
     position: relative; 
-    padding-top:var(--2_5);
+    padding-top: var(--2_5);
     padding-bottom: var(--2_5); 
     padding-left: var(--6);
     padding-right: var(--6); 
     color: #ffffff; 
-    background-color: rgb(37 99 235);
+    background-color: rgb(37 99 235) !important;
     align-items: center; 
     border-top-right-radius: var(--1);
     border-bottom-right-radius: var(--1); 
-    box-shadow: var(--0) var(--1) var(--1_5) -(var(--px)) rgba(0, 0, 0, 0.1), var(--0) var(--0_5) var(--1) -(var(--px)) rgba(0, 0, 0, 0.06); 
+    box-shadow: var(--0) var(--1) var(--1_5) calc(var(--px) * -1) rgba(0, 0, 0, 0.1), var(--0) var(--0_5) var(--1) calc(var(--px) * -1) rgba(0, 0, 0, 0.06); 
 }
 
 .searchBtn:hover {
-    box-shadow: var(--0) var(--2_5) var(--4) -(var(--0_5)) rgba(0, 0, 0, 0.1), var(--0) var(--1) var(--1_5) -(var(--0_5)) rgba(0, 0, 0, 0.05); 
+    box-shadow: var(--0) var(--2_5) var(--4) calc(var(--0_5) * -1) rgba(0, 0, 0, 0.1), var(--0) var(--1) var(--1_5) calc(var(--0_5) * -1) rgba(0, 0, 0, 0.05); 
```

Comment:
the `calc(val -1)` is harder to read then what you had before imo.

Reply:
@CobyPear I don't know how I did not notice this earlier but actually what I had there before was not working `-(var(--3))` was not registering as valid and was breaking the style. This way seems to work fine 👍

Reply:
ooh ok gotchya, works for me.